### PR TITLE
OP-782 | OPD Chart report to be translated

### PR DESCRIPTION
**Pull request by loique70**
I changed the the string key in the sql request inside the patient_opd_chart.jrxml

### CODE REVIEW

**Review by dbmalkovsky — APPROVED** [review]
LGTM

**Review by mwithi — CHANGES_REQUESTED** [review]
It will not work:
- missing .jasper
- the query should return just the key, and then translated within JasperReport
- remove all "KEY" from keys, they are unneded

please check my messages on Slack, thanks!

File: rpt_base/.gitignore
Comment (mwithi) [inline rpt_base/.gitignore:1]: please revert this change

File: rpt_base/patient_clinical_sheet_patsubreport.jasper
Comment (mwithi) [inline rpt_base/patient_clinical_sheet_patsubreport.jasper:1]: please, despite the subreport is recompiled automatically, revert this change

File: DataAdapter.jrdax
Comment (mwithi) [inline DataAdapter.jrdax:1]: Please don't push this file

File: rpt_base/patient_opd_chart.jrxml [hunk 1]
```diff
@@ -168,7 +169,7 @@ SELECT "ThirdDiagnosis" AS FIELD, CONCAT((SELECT DIS_DESC FROM OH_DISEASE WHERE
 				<textElement>
 					<font size="10"/>
 				</textElement>
-				<textFieldExpression><![CDATA[str($F{FIELD}) + ':']]></textFieldExpression>
+				<textFieldExpression><![CDATA[$F{FIELD}]]></textFieldExpression>
```
Comment (dbmalkovsky) [inline rpt_base/patient_opd_chart.jrxml]: is this statement even needed given the removal of the concatenation?

File: rpt_base/patient_opd_chart.properties [hunk 2]
```diff
@@ -6,18 +6,18 @@ OpdChart = OPD CHART
 
 version = version
 
-Id = ID
+IdKEY = ID
```
Comment (dbmalkovsky) [inline rpt_base/patient_opd_chart.properties]: should this have a colon like the other strings?

**Pull request merged**